src/app/workflow/info-tab/info-tab.component.html

```diff
@@ -204,22 +212,16 @@
             [canRead]="canRead"
             [canWrite]="canWrite"
             [isOwner]="isOwner"
+            [isPublished]="workflow?.is_published"
+            [workflowMode]="workflow?.mode"
+            [workflowId]="workflow.id"
           ></app-info-tab-checker-workflow-path>
           <!-- TODO: Remove once hosted workflows and Nextflow support checker workflows -->
           <div *ngIf="workflow?.mode === WorkflowType.ModeEnum.HOSTED || (isNFL$ | async)">
             <strong matTooltip="Currently disabled for Hosted Workflows and Nextflow Workflows">Checker Workflow</strong>: n/a
           </div>
-          <li *ngIf="!isPublic">
-            <strong [matTooltip]="modeTooltipContent">Mode</strong>: {{ workflow?.mode }}
-            <button
-              class="btn btn-link push-right"
-              type="button"
-              *ngIf="workflow?.mode === WorkflowType.ModeEnum.FULL && !workflow?.is_published"
```

Comment:
Why is this PR not just a simple `*ngIf="workflow?.mode === WorkflowType.ModeEnum.FULL && !workflow?.is_published && !workflow?`.isChecker"

Reply:
😨 ....bruh, I wasn't aware of this...